### PR TITLE
add multiple topics to a queue

### PR DESCRIPTION
**Pull request by samsamann**
This PR give us the ability to bind several topics on a queue. You can define a comma separated list of topics and subscriptions in the app.properties. The worker will notified for this events now.

`graviton.subscription=document.file.file.*,document.core.app.create`
`queue.bindKey=document.file.file.*,document.core.app.create`

### CODE REVIEW

File: src/main/java/com/github/libgraviton/workerbase/Worker.java [hunk 1]
```diff
@@ -84,14 +84,17 @@ private void connectToQueue() throws IOException {
...
+        String[] bindKeys = this.properties.getProperty("queue.bindKey").split(",");
         
         channel.exchangeDeclare(exchangeName, "topic", true);
         String queueName = channel.queueDeclare().getQueue();
 
-        channel.queueBind(queueName, exchangeName, bindKey);
-
-        LOG.info("[*] Subscribed on topic exchange '" + exchangeName + "' using binding key '" + bindKey + "'. Waiting for messages...");
+        for (String bindKey : bindKeys) {
+            bindKey = bindKey.trim();
```
Comment (wegenmic) [inline src/main/java/com/github/libgraviton/workerbase/Worker.java]: why is .trim() added here? the properties list separator should be a ',' without any whitespaces

File: src/main/java/com/github/libgraviton/workerbase/WorkerAbstract.java [hunk 2]
```diff
@@ -352,7 +352,7 @@ protected void registerWorker() throws Exception {
         
         for (String subscriptionKey: subscriptionKeys) {
             WorkerRegisterSubscription subObj = new WorkerRegisterSubscription();
-            subObj.setEvent(subscriptionKey);
+            subObj.setEvent(subscriptionKey.trim());
```
Comment (wegenmic) [inline src/main/java/com/github/libgraviton/workerbase/WorkerAbstract.java]: same as with Worker.java

File: src/main/java/com/github/libgraviton/workerbase/Worker.java [hunk 3]
```diff
@@ -84,14 +84,17 @@ private void connectToQueue() throws IOException {
         Channel channel = connection.createChannel();
 
         String exchangeName = this.properties.getProperty("queue.exchangeName");
-        String bindKey = this.properties.getProperty("queue.bindKey");
+        String[] bindKeys = this.properties.getProperty("queue.bindKey").split(",");
```
Comment (wegenmic) [inline src/main/java/com/github/libgraviton/workerbase/Worker.java]: I would suggest using Arrays.asList

File: src/main/java/com/github/libgraviton/workerbase/WorkerAbstract.java (hunk 2 above)
Comment (easy-da-man) [inline src/main/java/com/github/libgraviton/workerbase/WorkerAbstract.java]: subscriptionKeys.forEach(sk -> {
    //only with a fluent interface possible.
    //but this trim() looks suspicious. does that not mean if there is no whitespace in sk, you subscribe twice?
    WorkerRegisterSubscription wrs = new WorkerRegisterSubscription().setEvent(sk).setEvent(sk.trim());
});

----- Original Message -----

**Pull request merged**